README.md

```diff
@@ -1323,6 +1351,16 @@ Output
   'servicelevels': None
 }
 ```
+
+```yaml
+dataContractSpecification: 0.9.3
+id: uuid-custom
+info:
+  title: my_custom_imported_data
+  version: 0.0.1
+  description: Custom contract description
+
+```
```

Comment:
Nothing was imported here. Can you have a look?

Reply:
that was a bad example iuhaiuhaiuh
you are right